### PR TITLE
Added the ability to customize the feed's title

### PR DESCRIPTION
**Pull request by briced**

### CODE REVIEW

**Comment by KyleAMathews** [comment]
There's a lint error — https://travis-ci.org/gatsbyjs/gatsby/jobs/291842209

Could you run `npm run format` from the base of the repo? That should fix the lint error.

**Comment by briced** [comment]
Done. Sorry, I'm used to my projects with auto-format on commit (prettier + husky + lint-staged), and I totally forgot to run format

**Comment by KyleAMathews** [comment]
Thanks!

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 5f5af45c3c3a8582179005522860d2ccd4304f99

https://app.netlify.com/sites/using-glamor/deploys/59f0e918df99537566a3b9a2

File: packages/gatsby-plugin-feed/README.md [hunk 1]
```diff
@@ -76,7 +76,8 @@ plugins: [
               }
             }
           `,
-          output: '/rss.xml'
+          output: '/rss.xml',
+          feedTitle: 'My feed title'
```
Comment (nicholaswyoung) [inline packages/gatsby-plugin-feed/README.md]: This should be `title`. `feedTitle` is redundant.

File: packages/gatsby-plugin-feed/src/gatsby-ssr.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
         key={`gatsby-plugin-feed-${i}`}
         rel="alternate"
         type="application/rss+xml"
+        title={feedTitle}
```
Comment (nicholaswyoung) [inline packages/gatsby-plugin-feed/src/gatsby-ssr.js]: Again, `title`. I'll be changing this to match convention.

**Comment by nicholaswyoung** [comment]
After further research, I'm not sure this is even [valid HTML5 syntax.](https://developer.mozilla.org/en-US/docs/Correctly_Using_Titles_With_External_Stylesheets) The `title` attribute seems to be used primarily to reference alternate stylesheets, which the feed isn't. (I'd argue that the feed is persistent, and thus, the relevant title is embedded into the RSS document.)

Nor is `title` specified in the best practices guide published by the [RSS Advisory Board](http://www.rssboard.org/rss-profile).

I'll revert this until we can come to a better understanding, because while this validates, I'm not sure browsers will parse and respond to it properly, and in a manner the user expects.

**Comment by briced** [comment]
My reason for this PR was that https://tt-rss.org/ use the title for showing the list of feeds when parsing a website with multiple feeds. Without this, I was left with a choice between "application/rss+xml" and "application/rss+xml". Since many site I daily visit use the "title" attribute, I thought it was the common way to do this.

**Pull request merged**